scripts/beekeeper.pl

```diff
@@ -286,6 +288,15 @@ sub main {
 
     if($reset_job_id) { $queen->reset_job_by_dbID_and_sync($reset_job_id); }
 
+    if($reset_jobs_for_input_id and $self->{'analyses_pattern'}) {
```

Comment:
This should be moved down to be part of the "has_task" group at line 382. This is because that's after `@$list_of_analyses `has been built and checked to be sure that it is not empty. Otherwise, if the user provides an analyses_pattern that doesn't match anything, an empty list of analyses eventually makes its way to `AnalysisJobAdaptor::fetch_all_by_analysis_id_status()`, which causes a crash.